config/externalname.go

```diff
@@ -2081,6 +2078,9 @@ var NoForkExternalNameConfigs = map[string]config.ExternalName{
 	"aws_iot_thing_type": config.IdentifierFromProvider,
 	// IoT Topic Rules can be imported using the name
 	"aws_iot_topic_rule": config.NameAsIdentifier,
+	// IoT topic rule destinations can be imported using the arn
+	// arn:aws:iot:us-west-2:123456789012:ruledestination/vpc/2ce781c8-68a6-4c52-9c62-63fe489ecc60
+	"aws_iot_topic_rule_destination": TemplatedStringAsProviderDefinedIdentifier("arn:aws:iot:{{ .setup.configuration.region }}:{{ .setup.client_metadata.account_id }}:ruledestination/vpc/{{ .external_name }}"),
```

Comment:
What do you think about the relative merits of using just the provider-generated UUID suffix as the external name (as I've done here) vs the full arn (which is also the terraform id)? The arn is certainly longer, and more cumbersome, but also perhaps easier for new users to discover, since we don't currently have any way of documenting the expected format for an external name.

Reply:
I am okay with the above configuration and we can continue with this.